integration_test/testutil_test.go

```diff
@@ -205,48 +205,46 @@ func (it *ITest) WithStdin(r io.Reader) *ITest {
 	return it
 }
 
-// RunOrFail runs the krew command and fails the test if the command returns an error.
-func (it *ITest) RunOrFail() {
-	it.t.Helper()
-	if err := it.Run(); err != nil {
-		it.t.Fatal(err)
-	}
-}
-
-// Run runs the krew command.
-func (it *ITest) Run() error {
+// Run runs the krew command, and returns its combined output even when
+// it fails.
+func (it *ITest) Run() ([]byte, error) {
 	it.t.Helper()
 
 	cmd := it.cmd(context.Background())
+	if it.stdin != nil {
+		cmd.Stdin = it.stdin
+	}
 	it.t.Log(cmd.Args)
 
+	var b bytes.Buffer
+	cmd.Stdout = &b
+	cmd.Stderr = &b
+
 	start := time.Now()
 	if err := cmd.Run(); err != nil {
-		return errors.Wrapf(err, "krew %v", it.args)
+		out := b.Bytes()
+		return out, errors.Wrapf(err, "krew %v: %v, %s", it.args, err, string(out))
 	}
-
 	it.t.Log("Ran in", time.Since(start))
-	return nil
+	return b.Bytes(), nil
```

Comment:
This can be simpler:
```go
err := cmd.Run()
it.t.Log("Ran in", time.Since(start))
return b.Bytes(), errors.Wrapf(err, "krew %v: %v, %s", it.args, err, string(out))
```
WDYT?

Reply:
yeah 👍  seems like it except we need to call `b.Bytes()` only once